inference/enterprise/parallel/README.md

```diff
@@ -0,0 +1,32 @@
+# Parallel Robofolow Inference server
+
+Introducing the highly concurrent implementation of Roboflow Inference Server! This version of the server accepts and processes requests asynchronously, running the web server, preprocessing, auto batching, inference, and post processing all in separate threads to increase server fps throughput. Separate requests to the same model will be batched on the fly as allowed by `$MAX_BATCH_SIZE`, and then response handling will occurr independently. Images are passed via python's SharedMemory module to maximize throughput.
```

Comment:
Please replace the first sentence with something like:

```
The Roboflow Inference Server supports concurrent processing.
```

We should be direct in the first sentence. It is the hook.

Learn more: https://jamesg.blog/2023/12/03/first-sentences/

Reply:
> server fps throughput

FPS needs capitals.

Reply:
> Images are passed via python's SharedMemory module to maximize throughput.

"python" needs a capital letter.